docs/0.5/references/cli/splinter-circuit-purge.1.md

```diff
@@ -0,0 +1,85 @@
+% SPLINTER-CIRCUIT-PURGE(1) Cargill, Incorporated | Splinter Commands
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+NAME
+====
+
+**splinter-circuit-purge** — Submits a request to purge the specified circuit.
+
+SYNOPSIS
+========
+**splinter circuit purge** \[**FLAGS**\] \[**OPTIONS**\] CIRCUIT-ID
+
+DESCRIPTION
+===========
+Request to purge a circuit by specifying the circuit ID of the circuit to be
+removed from the node's storage. A circuit is only available to be purged
+if it has already been disbanded and are only available locally. Disbanding a
```

Comment:
The 2nd part of this sentence ("...and are only available locally") should be revised; it's not clear what is being talked about.

Reply:
This paragraph is currently edited in a PR in splinter: https://github.com/Cargill/splinter/pull/1201/commits/1f89421e5cf2b6b3a1736e7e53a8d90124edecfc, I will update that PR with this feedback. This will help keep the man pages consistent between the repos. I'll be putting up a PR in this repo once those updates are merged to update the man pages in this repo, which will fix the part commented on here.